### PR TITLE
Add pagination to tokens page

### PR DESCRIPTION
**Pull request by Lerek**

### CODE REVIEW

**Comment by Riddlerrr** [comment]
The code looks good to me 👍

Would be nice to have a screenshot in PR description if it related to frontend changes to show the new version.
It also will help to test the changes.

**Comment by Lerek** [comment]
@Riddlerrr thanks, and sure, here it is:
image

File: app/controllers/tokens_controller.rb [hunk 1]
```diff
@@ -81,6 +81,25 @@ def fetch_contract_details
 
   private
 
+    def set_tokens
```
Comment (matjacdev515) [inline app/controllers/tokens_controller.rb:84]:
```ruby
def tokens
    @tokens ||=
      policy_scope(Token)
      .with_attached_logo_image
      .page(params[:page])
      .per(TOKENS_PER_PAGE)
  end
```

And use `tokens` method where it is needed, without calling `set_tokens` separately. What do you think?

**Comment by Lerek** [comment]
@matjacdev515 done, could you please take another look? :)

**Review by aquabu — APPROVED** [review]

**Pull request merged**